### DISCUSSION TITLE
The Pull Shark Badge is confusing

### DISCUSSION THREAD

**Discussion by mahmoud-moursy** [community/community#37716] [Other Feature Feedback, Questions, & Ideas]
The badge design itself, with the 2x, makes the user think that you get one per PR -- and the UI only showed me 2 PRs as well which made me all the more confused. I think the badge should be redesigned.

**Comment by antonblr** [comment]
It kind of outlined in the badge details that it is given for the 2nd, 16th etc pull requests merged. 

<img width="346" alt="Untitled" src="https://user-images.githubusercontent.com/44246099/198843631-7646f251-7e5b-41cb-a821-3fbf30a6b6e2.png">


But I see how it can be not that straight forward to grasp at the very beginning. Perhaps they could have signs like `bronze`, `silver`, `gold` instead of `x2`, `x3`, `x4`, but that's more of a nit to me.

**Comment by Ezequiel-Mathias** [comment]
yes, I was in the same dilemma, but I found out that it counts by merging, if you made two soon you will have two, a very interesting repository that explains about the emplamas and how to acquire them is this one here:
 
https://github.com/Schweinepriester/github-profile-achievements

Give it a go, I think it will help you.

and if you can mark my comment as the answer to your "question" I will be grateful 😀

**Comment by aik099** [comment]
I'm on GitHub for a long time and I have made a lot of Pull Requests since then. The badge however shows `x2` as if I've made 2 pull requests in total. Very confusing.

**Reply by david-engelmann** [reply]
I had a similar issue with commits not being attributed to my account. Make sure all the emails you've committed under are attributed to your account, ie. aik.bold@gmail.com, alex@aik.ninja and aik099@users.noreply.github.com

**Comment by cx0y** [comment]
It's confusing! 😂
you'll get! 
x1 - 2 PRs
x2 - 16 PRs
x3 - 128 PRs
x4 - 1024 PRs

**Comment by mohamedzhioua** [comment]
in the link below you can see how you can get every achievment badge :
  "https://github.com/Schweinepriester/github-profile-achievements"

**Comment by anmorgan24** [comment]
Hi @T-O-R-U-S, you're right, it can be a bit confusing. I had this same exact question! Luckily, there's a few fantastic resources outlining how exactly to get various achievements badges (including the Pull Shark one!).

I really like this repo (which was also suggested by another user): https://github.com/Schweinepriester/github-profile-achievements

You'll see there that you need to merge 2 PRs to get the badge, but if you need additional support to figure out how exactly to do this, this is a helpful tutorial I found on YouTube: https://www.youtube.com/watch?v=wBW-vjM7ccA

I hope this helps!

**Comment by ashish5kmax** [comment]
![ans1](https://github.com/user-attachments/assets/f3a9c77b-78f2-499a-858c-a6878f838e66)
![ans2](https://github.com/user-attachments/assets/3437e02a-8b56-4048-8604-83ed716f0509)

* The understanding form the pictures I got is that it's the amount of pull requests you open not closed as you can see from the above pics.
So, basically :-

1) 2 for getting the pull shark badge.
2) 16 bronze,
3) 128 silver,
4) 1024 gold.

* I hope your question is answered.

* Thank you.

**Comment by Hilal-11** [comment]
> The badge design itself, with the 2x, makes the user think that you get one per PR -- and the UI only showed me 2 PRs as well which made me all the more confused. I think the badge should be redesigned.

Pull Shark is earned by having your pull requests merged — the tiers just track cumulative count, not anything confusing:

Default (x1): 2 merged PRs
Bronze (x2): 16 merged PRs
Silver (x3): 128 merged PRs
Gold (x4): 1,024 merged PRs
